__tests__/html/activityStatusTelemetry.sendingToSendFailed.html

```
@@ -0,0 +1,75 @@
+            onTelemetry: (event) => {
+              telemetryEventTracking.push(event);
+            },
+            styleOptions: {
+              sendTimeout: 5000
+            }
+          },
+          document.getElementById('webchat')
+        );
+
+        await pageConditions.uiConnected();
+
+        const sendMessage1stAttempt = await directLine.emulateOutgoingActivity('Egress message 1, sending to send failed');
+        sendMessage1stAttempt.rejectPostActivity(new Error('artificial error for egress message 1'));
+
+        const sendMessage2ndAttempt = await directLine.emulateOutgoingActivity('Egress message 2,  sending to send failed');
+        sendMessage2ndAttempt.rejectPostActivity(new Error('artificial error for egress message 2'));
+
+        await pageConditions.became(
+          'failed to send message',
+          () => {
+            return pageElements.activityStatuses()[0]?.innerText === 'Send failed. Retry.' &&
+            pageElements.activityStatuses()[1]?.innerText === 'Send failed. Retry.'
+          },
+          1000
+        );
+
+        const expectedEventName = 'send-status:change';
+        let lastRecord = telemetryEventTracking.pop();
+        expect(lastRecord.data).toEqual({
+          currentStatus: 'send failed',
+          previousStatus: 'sending',
+          clientActivityID: expect.any(String),
+          type: 'message',
+          key: expect.any(String),
+          hasAttachment: 'false'
+        });
+        expect(lastRecord.name).toEqual(expectedEventName);
+        
+        lastRecord = telemetryEventTracking.pop();
+        expect(lastRecord.data).toEqual({
+          currentStatus: 'send failed',
+          previousStatus: 'sending',
+          clientActivityID: expect.any(String),
+          type: 'message',
+          key: expect.any(String),
+          hasAttachment: 'false'
+        });
+        expect(lastRecord.name).toEqual(expectedEventName);
+      },
+      { ignoreErrors: true });
```

```suggestion
      });
```

---

we need to ignore the errors as we artificially injecting errors,

---

You caught me. 🤣